readthedocs/basic/updates.rst

```diff
@@ -16,7 +16,7 @@ For that, you can use **events**.
     .. code-block:: python
 
         import logging
-        logging.basicConfig(format='[%(levelname) 5s/%(asctime)s] %(name)s: %(message)s',
+        logging.basicConfig(format='[%(levelname) %(asctime)s] %(name)s: %(message)s',
```

Comment:
It's intentional.

```python
>>> obj = {"levelname": "INFO", "asctime": "2024", "name": "telethon", "message": "some log"}
>>> '[%(levelname) 5s/%(asctime)s] %(name)s: %(message)s' % obj
'[ INFO/2024] telethon: some log'
>>> obj["levelname"] = "ERROR"
>>> '[%(levelname) 5s/%(asctime)s] %(name)s: %(message)s' % obj
'[ERROR/2024] telethon: some log'
```

The ` 5s` is right-aligning it to the right with spaces.

Reply:
On my console, this didn't happen, I tried both with 5s/ and whiteout it, and only difference was the console actually printed 5s/ between the error level and time

Reply:
Weird. Perhaps the `logging` module behaves differently. What Python version are you using?

Reply:
Using Python 3.11.9
Code:
```
logging.basicConfig(
    format="[%(levelname)s - %(asctime)s] %(name)s: %(message)s", level=logging.DEBUG
)
```
Output:
[INFO - 2024-09-11 11:48:05,330] root: Sample Log

Code: 
```
logging.basicConfig(
    format="[%(levelname)s - 5s/%(asctime)s] %(name)s: %(message)s", level=logging.DEBUG
)
```
Output:
[INFO - 5s/2024-09-11 11:50:00,738] root: Sample Log

Reply:
> ```python
> obj = {"levelname": "INFO", "asctime": "2024", "name": "telethon", "message": "some log"}
> >>> '[%(levelname) 5s/%(asctime)s] %(name)s: %(message)s' % obj
> ```

This code works, but this string bahavior seems not working when in logging.

Reply:
Was this the intention?
Code:
```
logging.basicConfig(
    format="[%(levelname)-7s/%(asctime)s] %(name)s: %(message)s", level=logging.DEBUG
)
```
Output:
```
[DEBUG  /2024-09-11 12:19:35,839] root: Sample Debug
[INFO   /2024-09-11 12:19:35,839] root: Sample Info
[WARNING/2024-09-11 12:19:35,839] root: Sample Warning
[ERROR  /2024-09-11 12:19:35,839] root: Sample Error
```

Source:
[printf-style String Formatting](https://docs.python.org/3/library/stdtypes.html#old-string-formatting)

Reply:
Weird that there's left-adjusted but not right-adjusted. Well, doesn't really hurt to merge this I suppose.